dhall-docs/dhall-docs.cabal

```diff
@@ -97,3 +113,21 @@ Test-Suite doctest
         doctest    >= 0.7.0   && < 0.18
     Other-Extensions: OverloadedStrings RecordWildCards
     Default-Language: Haskell2010
+
+Test-Suite tasty
+    Type: exitcode-stdio-1.0
+    Hs-Source-Dirs: tasty
+    Main-Is: Main.hs
+    Build-Depends:
+        base                   ,
+        containers             ,
+        dhall-docs             ,
+        HaXml        >= 1.25.5 ,
+        path                   ,
+        path-io                ,
+        pretty       >= 1.1.1.1,
+        tasty        <  1.4    ,
```

Comment:
basically because I copied that from other test setups.

Regarding other lower bounds: basically because I know my code works in those version. I can add semantic upper bounds to those

Reply:
Okay! I often don't use upper bounds for test suite dependencies because it's fairly inconsequential when the testsuite breaks due to a change in a dependency.